RiotSwiftUI/Modules/Room/LocationSharing/View/LocationSharingUserMarkerView.swift

```diff
@@ -27,19 +27,35 @@ struct LocationSharingUserMarkerView: View {
     
     @State private var frame: CGRect = .zero
     
+    private var usernameColorGenerator: UserNameColorGenerator {
```

Comment:
You can reuse `@Environment(\.theme) private var theme: ThemeSwiftUI` here and update with the convenient `Themable` method suggested above.